examples/tests/test_simple.py

```diff
@@ -1,7 +1,5 @@
 # This file is used to test if the test function collector works.
-import pytest
-
-from examples import sample_compare_integers, sample_convert_string_to_int, sample_compare_two_int_lists
+from examples.simple import sample_compare_two_int_lists, sample_compare_integers, sample_convert_string_to_int
```

Comment:
You are using our examples in your test code; You should create a copy of the samples you need and add them to the resources.

Reply:
(I hope I understood your comment correctly) 
test_simple is not a unit test for our project, it is a sample unit test to be traced.